### PR TITLE
[libcalamaresui] sync naming change

### PR DESCRIPTION
**Pull request by demmm**
Branding.h moved the style/color naming, Branding.cpp needs to be in sync
add the changes to the default branding.desc & calamares-sidebar.qml

### CODE REVIEW

File: src/branding/default/branding.desc [hunk 1]
```diff
@@ -182,8 +182,8 @@ images:
 style:
    sidebarBackground:    "#292F34"
    sidebarText:          "#FFFFFF"
-   sidebarTextSelect:    "#292F34"
-   sidebarTextHighlight: "#D35400"
+   sidebarTextCurrent:    "#292F34"
+   SidebarBackgroundCurrent: "#D35400"
```
Comment (dalto8) [inline src/branding/default/branding.desc]: The casing is inconsistent here.

Comment (demmm) [reply src/branding/default/branding.desc]: This is to make the same as for Branding.h https://github.com/calamares/calamares/commit/a7d3630e42b7c001fc6a209a56e3baa7e25f575d#diff-ace2b49f072feb0bbaeea9f8e4197ffb50b3dbfe14a3327226bebef8e726ad43R79

Comment (demmm) [reply src/branding/default/branding.desc]: Sorry, missed the mistake twice.....

File: src/calamares/calamares-sidebar.qml [hunk 2]
```diff
@@ -51,7 +51,7 @@ Rectangle {
                 Text {
                     anchors.verticalCenter: parent.verticalCenter;
                     anchors.horizontalCenter: parent.horizontalCenter;
-                    color: Branding.styleString( index == ViewManager.currentStepIndex ? Branding.SidebarTextCurrent : Branding.SidebarText );
+                    color: Branding.styleString( index == ViewManager.currentStepIndex ? Branding.SidebarTextSelect : Branding.SidebarText );
```
Comment (adriaandegroot) [inline src/calamares/calamares-sidebar.qml:54]: This looks backwards to me. You just introduced the **new** name SidebarTextCurrent.

**Comment by adriaandegroot** [comment]
What is this fixing, exactly? There are multiple names in play here -- they might not be consistent, which is certainly **confusing**, but the change seems to confuse them "the other way" from what they were.

- There's the enum **identifiers**, visible in C++; that's `enum StyleEntry` in `Branding.h`
- There are the enum names visible to Qt, generated through `Q_ENUM`; these names are the same as the identifiers
- There are the keys used in the configuration files

Because of the (kinda derpy) way that the Branding object loads the strings from the *style* key in the config file, the keys need to be the same as the identifiers.

(It might have been part of the plan to break all the `branding.desc` files for 3.3, introducing the new and somewhat-more-consistent names. That would be something that can be extended a bit, checking that the style names that are **used** in the `branding.desc` file are actually valid)

**Comment by demmm** [comment]
With the change in Branding.h, I could not get calamares-seidebar.qml to show correct colors with what is currently set for colors in branding.desc.  So I tried to make Branding.cpp & Branding.h use the same terms (colors now show correctly locally).

But I probably misunderstand what the todo removal/commit in Branding.h accomplished, so I will keep the changes locally only.

**Pull request closed without merging**